editor/questionbankcontroller.js

```diff
@@ -148,6 +151,7 @@ angular.module('createquestionapp', [])
       ecEditor.getService('assessment').getQuestions(data, function (err, resp) {
         if (!err) {
           $scope.questions = resp.data.result.items;
+          savedQuestions = $scope.questions;
```

Comment:
Why do you need savedQuestions  variable when you have $scope.questions?

Reply:
Because $scope is getting overridden the closure is failing. If we use a local variable, it is always preserved.